registry/infrastructure/repositories/organization_repository.py

```diff
@@ -162,15 +156,16 @@ def _update_org_status(self, org_workflow_model, status, username):
         org_workflow_model.OrganizationReviewWorkflow.updated_on = current_utc_time
         org_workflow_model.OrganizationReviewWorkflow.updated_by = username
 
-    def add_group(self, groups, org_uuid):
+    def add_group(self, groups, org_uuid, username):
         organizations = self.get_org_for_uuid(org_uuid)
         if len(organizations) > 0:
             latest_org_record = None
             for org in organizations:
                 if org.OrganizationReviewWorkflow.status == OrganizationStatus.DRAFT.value:
                     latest_org_record = org
             if latest_org_record is None:
-                pass
+                published_record = organizations[0]
```

Comment:
please verify if org[0] is always published, also please make the code more readable